### ISSUE TITLE
Cooperative 3.0

### ISSUE THREAD

**Issue by DanM3rcurius** [SecretDecks/Documentation#110]
## Goals

- [ ] Make contributing easy for cooperative members. 
- [ ] GUI (for collaboration, tracking, validation, voting, polling, discussions) on website/-app (secretdecks.com). 
- [ ] Use GitHub as data backend and Discord as communication "backend". 
- [ ] Automate task tracking and administrative overhead where possible with GitHub actions.

## Specifications

TBD

## Journal / Timeline 
- May 2020: found that many struggle using github and discord; our audience needs to be picked up somehow else
- June 2020: started brainstorm about the website being the 3.0 web portal to the cooperative
- September 2020: @tamagit80 started setup OAuth #109

**Comment by stephen-rowan** [comment]
In ZenHub Cooperative 3.0 should probably be a Project broken into smaller epics ?

**Comment by tamagit80** [comment]
> In ZenHub "Cooperative 3.0" should probably be a Project broken into smaller epics ? The Goals you have listed would translate better to Epics - so SD need to define high level tasks as Epics. Every Issue would would then help complete an Epic.

I totally agree with that. Would be easier to track and to split responsabilities/roles. We could also create groups for every epic->issue.

**Comment by DanM3rcurius** [comment]
Dear Stephen and GL,
you both bring up very good points!
I thought we start with one project (SecretDecks) and break it down into epics. While we do this we could define and answer the following:

- When is sth an epic or a separate project?
- Will there be sprints for each project? Meaning someone in two projects would do a "double" sprint?
- How should inter project communication look like?
- How to mitigate having many projects started but not finished?
- Is it possible to do resource planning across multiple projects?

Anyhow we should have an agile kickoff meeting to decide on starting parameters.
For this we need to collect parameters such as:
- When is an item an issue, task, epic or project?
- Whats the information flow between projects?
- Dashboard (overview) of all projects?

What else?

Sent with [ProtonMail](https://protonmail.com/) Secure Email.

‐‐‐‐‐‐‐ Original Message ‐‐‐‐‐‐‐

**Comment by stephen-rowan** [comment]
Hi Dan :)

## When is sth an epic or a separate project?

A **project** delivers a major outcome - e.g. Website Launch, Software release - it always has a clear scope, end date and deliverables. Very much like a Catalyst proposal. A project will have milestones which are analogous to Epics.

An **Epic** is a major component of a project - e.g. Complete Website design, complete software feature - it typically has dependencies - e.g. Issues.

An **Issue** is a discrete task that forms part of an Epic.

 ## Will there be sprints for each project?

No - Sprints are set at the high level for the entire workspace - see this screenshot - one sprint (period) rules
 
![2021-09-29](https://user-images.githubusercontent.com/25156451/135301185-dc4426fd-1563-45eb-8d28-06ca1c6c840c.png)

In this screenshot - I am assigned to two Epics (but they could be Issues) in two different projects - but in the same workspace sprint ( Sep 27 - Oct 3) 

### Meaning someone in two projects would do a "double" sprint?

No - see above

###  How should inter project communication look like?

Each member will be able to see all their assignments across all projects (using the assignment filter)

### How to mitigate having many projects started but not finished?

By only having major projects - SecretDecks needs to define its high level projects (deliverables).

### Is it possible to do resource planning across multiple projects? 

Yes - see above - it is also possible to add resource estimates in ZenHub

### Anyhow we should have an agile kickoff meeting to decide on starting parameters. 

Yes

**Comment by DanM3rcurius** [comment]
Dear  Stephen,

thanks for the great answers! 

I phrased my question quite bad.

> ## Will there be sprints for each project?
> No - Sprints are set at the high level for the entire workspace - see this screenshot - one sprint (period) rules all
> 

I meant to inquire if every teamworkspace can set its own sprint length? For example copywriting or marketing might be more comfortable with a shorter sprint than engineering? Or will that create too much friction during handoffs? I can't see why.

**Comment by stephen-rowan** [comment]
> 
> 
> Dear Stephen,
> 
> thanks for the great answers!
> 
> I phrased my question quite bad.
> 
> > ## Will there be sprints for each project?
> > No - Sprints are set at the high level for the entire workspace - see this screenshot - one sprint (period) rules all
> 
> I meant to inquire if every teamworkspace can set its own sprint length? For example copywriting or marketing might be more comfortable with a shorter sprint than engineering? Or will that create too much friction during handoffs? I can't see why.

Will have to look into that - we might have to create separate projects or workspaces to accommodate different sprint lengths

**Comment by DanM3rcurius** [comment]
> > Dear Stephen,
> > thanks for the great answers!
> > I phrased my question quite bad.
> > > ## Will there be sprints for each project?
> > > No - Sprints are set at the high level for the entire workspace - see this screenshot - one sprint (period) rules all
> > 
> > 
> > I meant to inquire if every teamworkspace can set its own sprint length? For example copywriting or marketing might be more comfortable with a shorter sprint than engineering? Or will that create too much friction during handoffs? I can't see why.
> 
> Will have to look into that - we might have to create separate projects or workspaces to accommodate different sprint lengths

not urgent. was just curious and seeing how far we can go with this.

**Comment by DanM3rcurius** [comment]
@dgarey please look at what was setup here, to get an understanding of the big picture; where we want to go with DisCo 3.0
